app/assets/javascripts/application.js

```diff
@@ -1 +1,18 @@
 //= require 'uswds/uswds'
+//= require 'jquery'
+//= require 'owl-carousel2'
+
+(function(window, $) {
+  'use strict';
+
+  $('.owl-carousel').owlCarousel({
```

Comment:
Where does rails put this JS file? Point being... does this need a `$(document).ready()`?